### PR TITLE
Fix/22174 clarify installtion overview os options

### LINKED ISSUE #22174: Installations Overview: Make it obvious we have instructions on how to set up Windows computers to do TOP
https://www.theodinproject.com/courses/foundations/lessons/installation-overview#os-options

Have been noticing people thinking they NEED a mac to Odin so they never start, and say things like "I wish I could do this without a mac/you had options to make it work for Windows people"

It was briefly discussed in Discord. People see **Mac** and **Linux** and nothing about **Windows** in bold at first glance (yea, they should be reading, but I guess a lot don't)

Adjust OS Options section to make it clear just because you have a windows computer does NOT mean you can't do odin. People seem to think if you must have a mac, or there aren't options (VM/Dual Boot) that can make a windows computer work. 

The words "OS/operating system" and "Linux"  "VM Dual boot" and installation stuff is often foreign to a lot of people who never have messed with this type of thing and just start using their computer right out of the box. So making it clear that the VM DualBoot section is for Windows people would be helpful! (possibly by changing the "**Virtual Machines and Dual-Boots**" heading in this section to "**Windows**" and adjust that section accordingly.

### PR DESCRIPTION
**Pull request by wise-king-sullyman**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

Revise the installations overview lesson in foundations to clarify that people using windows are able to do TOP without having to buy a new computer.

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

Close #22174

### CODE REVIEW

File: foundations/installations/installation_overview.md [hunk 1]
```diff
@@ -24,15 +24,24 @@ At the end of the next lesson, you'll be up and running with many of the tools y
...
 
 [Linux](https://en.wikipedia.org/wiki/Linux) is a free and open-source operating system that works well with all programming languages. Most development tools are written to work natively with Linux. Your tools will likely be updated more often, have more information available for troubleshooting, and just plain run better on Linux. We'll be using Ubuntu, one of the most popular and user-friendly distributions available, or the lighter weight alternative Xubuntu. **If you don't use a Mac, we recommend that you use Linux.** It's that simple.
 
-#### Virtual Machines and Dual-Boots
+</details>
+
+<details markdown="block">
+<summary class="dropDown-header">Windows</summary>
+
+Windows is **not natively supported** by the odin project on this website or our discord server; however, if you're currently using windows you can use either a virtual machine or dual booting to keep your windows install while creating your development environment in Linux.
```
Comment (twalton83) [inline foundations/installations/installation_overview.md:44]: Can we capitalize the TOP name and “windows”?

File: foundations/installations/installation_overview.md [hunk 2]
```diff
@@ -24,15 +24,24 @@ At the end of the next lesson, you'll be up and running with many of the tools y
 
 ### OS Options
 
-#### Mac
+<details markdown="block">
+<summary class="dropDown-header">Mac</summary>
```
Comment (xandora) [inline foundations/installations/installation_overview.md:28]:
```suggestion
<summary class="dropDown-header">macOS</summary>
```

**Review by twalton83 — COMMENTED** [review]
Just a few small things but other than that I love it.

File: foundations/installations/installation_overview.md (hunk 1 above)
Comment (xandora) [inline foundations/installations/installation_overview.md:44]:
```suggestion
Windows is **not natively supported** by The Odin Project, or on our Discord server; however, if you're currently using Windows you can use either a virtual machine or dual boot to keep your Windows install while creating your development environment in Linux.
```

**Review by twalton83 — APPROVED** [review]
I went ahead and committed the suggestions, but this is good to go for me!

**Pull request merged**